examples/research_projects/mulit_token_textual_inversion/README.md

```diff
@@ -1,4 +1,7 @@
-## Multi Token Textual Inversion
+## [Deprecated] Multi Token Textual Inversion
```

Comment:
@isamu-isozaki would this be ok for you given that we have multi token now natively supported?

Reply:
@patrickvonplaten Oh yup no worries go ahead! Looks awesome